baselines/webworker.generated.d.ts

```diff
@@ -1530,363 +1652,53 @@ declare var XMLHttpRequestUpload: {
-    new(): AbortController;
-};
-
-interface AbortSignal extends EventTarget {
-    readonly aborted: boolean;
-    onabort: (ev: Event) => any;
-}
-
-interface EventSource extends EventTarget {
-    readonly CLOSED: number;
-    readonly CONNECTING: number;
-    readonly OPEN: number;
-    onerror: (evt: MessageEvent) => any;
-    onmessage: (evt: MessageEvent) => any;
-    onopen: (evt: MessageEvent) => any;
-    readonly readyState: number;
-    readonly url: string;
-    readonly withCredentials: boolean;
-    close(): void;
-}
-
-declare var EventSource: {
-    prototype: EventSource;
-    new(url: string, eventSourceInitDict?: EventSourceInit): EventSource;
-};
-
-interface EventSourceInit {
-    readonly withCredentials: boolean;
-}
-
 declare type EventListenerOrEventListenerObject = EventListener | EventListenerObject;
 
 interface DecodeErrorCallback {
     (error: DOMException): void;
 }
+
 interface DecodeSuccessCallback {
     (decodedData: AudioBuffer): void;
 }
+
 interface ErrorEventHandler {
-    (message: string, filename?: string, lineno?: number, colno?: number, error?: Error): void;
+    (event: Event | string, source?: string, fileno?: number, columnNumber?: number, error?: Error): void;
 }
+
 interface ForEachCallback {
-    (keyId: any, status: MediaKeyStatus): void;
+    (keyId: Int8Array | Int16Array | Int32Array | Uint8Array | Uint16Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array | DataView | ArrayBuffer | null, status: MediaKeyStatus): void;
 }
+
 interface FunctionStringCallback {
     (data: string): void;
 }
+
 interface NotificationPermissionCallback {
     (permission: NotificationPermission): void;
 }
+
 interface PositionCallback {
     (position: Position): void;
 }
+
 interface PositionErrorCallback {
     (error: PositionError): void;
 }
-declare var onmessage: (this: DedicatedWorkerGlobalScope, ev: MessageEvent) => any;
+
+declare var onmessage: ((this: DedicatedWorkerGlobalScope, ev: MessageEvent) => any) | null;
 declare function close(): void;
 declare function postMessage(message: any, transfer?: any[]): void;
+declare function dispatchEvent(evt: Event): boolean;
 declare var caches: CacheStorage;
 declare var isSecureContext: boolean;
 declare var location: WorkerLocation;
-declare var onerror: (this: DedicatedWorkerGlobalScope, ev: ErrorEvent) => any;
+declare var onerror: ((this: DedicatedWorkerGlobalScope, ev: ErrorEvent) => any) | null;
 declare var performance: Performance;
 declare var self: WorkerGlobalScope;
-declare function msWriteProfilerMark(profilerMarkName: string): void;
 declare function createImageBitmap(image: ImageBitmap | ImageData | Blob, options?: ImageBitmapOptions): Promise<ImageBitmap>;
 declare function createImageBitmap(image: ImageBitmap | ImageData | Blob, sx: number, sy: number, sw: number, sh: number, options?: ImageBitmapOptions): Promise<ImageBitmap>;
+declare function msWriteProfilerMark(profilerMarkName: string): void;
```

Comment:
Is `msWriteProfilerMark` not tagged as an IE/MS-only thing? It looks like one.

Reply:
the input file has bugs as well :D. I did talk to the authors in the Edge team and they are aware of some of these untagged declarations, and plan to address them.

I was planning on filtering out all "ms*" and "MS*" declarations, but will get that in a separate change.